json2html/jsonconv.py

```diff
@@ -23,6 +23,12 @@ def convert(self,**args):
 		'''
 		convert json Object to HTML Table format
 		'''
+		global table_attributes
+		table_attributes = ''
+		if 'table_attributes' in args:
+			table_attributes = args['table_attributes']
+		else:
+			table_attributes = "<table border=\"1\">"
```

Comment:
Please check once, this will make the tag `<table <table border="1">>` as you have used ```table_init_markup = "<table %s>" %(table_attributes)
a=a+ table_init_markup```

Please replace this with `table_attributes = 'border="1"'`